File: Conversao Maiusculo e Minusculo/maiusminus.scala
import scala.io.StdIn

object Main {
  def main(args: Array[String]): Unit = {
    print("Digite um texto .....: ")
    val texto = StdIn.readLine()

    val textoMaiusculo = texto.toUpperCase
    val textoMinusculo = texto.toLowerCase

    print("Texto em maiusculo ..: ")
    println(textoMaiusculo)

    print("Texto em minusculo ..: ")
    println(textoMinusculo)
  }
}

File: Conversao Maiusculo e Minusculo/maiusminus.dart
import 'dart:io';

void main() {
  stdout.write('Digite um texto .....: ');
  String? texto = stdin.readLineSync();

  String textoMaiusculo = texto!.toUpperCase();
  String textoMinusculo = texto.toLowerCase();

  stdout.write('Texto em maiusculo ..: ');
  print(textoMaiusculo);

  stdout.write('Texto em minusculo ..: ');
  print(textoMinusculo);
}
